designHashset.py

```diff
@@ -0,0 +1,66 @@
+## // Time Complexity :average time complexity O(1) and in worst case O(C) length of chain 
+# // Space Complexity :O(n)
+# // Did this code successfully run on Leetcode : yes
+# // Any problem you faced while coding this : no
+
+
+class designHashSet:
+    def __init__(self):
+        self.totalbucket=1500 #random bucket size based on keys and number of call that can be made
```

Comment:
Try to chose `totalbucket` size strategically and implement double hashing as well.